### PR TITLE
Contribute cloudant plugin

### PR DESCRIPTION
**Pull request by killuazhu**
Add one more plugin to detect [Cloudant tokens](https://cloud.ibm.com/docs/services/Cloudant?topic=cloudant-authentication).

### CODE REVIEW

File: detect_secrets/plugins/cloudant.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        headers = {'Content-type': 'application/json'}
+        request_url = 'https://{hostname}:' \
+            '{token}' \
+            '@{hostname}.' \
+            'cloudant.com'.format(
+                hostname=hostname,
+                token=token,
+            )
+
+        response = requests.get(
```
Comment (KevinHock) [inline detect_secrets/plugins/cloudant.py]: Can you do the specific exception thing here?

Rest of the PR ✅ ⛵️ 🚢 👍

**Comment by killuazhu** [comment]
> I didn't realize this in the other PR, but it is a little confusing that the numbers referenced in your commit messages map to unrelated issue/PR #'s, presumably b/c they're internal numbers.

Yes, we use Github Enterprise internally, which might mess up with the PR. I can remove the PR id from commit message to avoid confusion.

File: detect_secrets/plugins/cloudant.py (hunk 1 above)
Comment (killuazhu) [reply detect_secrets/plugins/cloudant.py]: Comment addressed.

**Comment by killuazhu** [comment]
Pushed again with updated commit message.

**Comment by killuazhu** [comment]
Retrigger Travis build.

**Pull request merged**